docs/overview_and_guide.rst

```diff
@@ -150,6 +151,15 @@ After account creation, launch Raiden with the path of your keystore supplied::
 
     raiden --keystore-path ~/.local/share/io.parity.ethereum/keys/test
 
+.. _using_rpc-endpoint:
+
+Using --eth-rpc-endpoint/Infura
+*******************************
+
+To use Raiden with with an rpc-endpoint provided by an Infura Ethereum node sign up with `Infura <https://infura.io/>`_ to get an API token. After that you directly start using Raiden on Ropsten::
```

Comment:
Too many `with`s.

and also I would say: "After that you can start using Raiden directly on Ropsten"